Auth/AuthAPI.py

```diff
@@ -1,7 +1,44 @@
-# Score API here
 from flask import Blueprint
+from flask import request, jsonify, current_app
 import sys
 from db import db
+from db import credentials
+import jwt
 sys.path.append("../")
 
 auth_api = Blueprint("auth", __name__)
+
+@auth_api.route('/register', methods=['POST'])
+def register():
+    username = request.args.get('username')
+    pw = request.args.get('passwordHash')
+    credentials.append({"username": username, "passwordHash": pw})
```

Comment:
Defensive programming: Check that input is not "" or None to ensure that its a valid input